pkg/storage/internalstorage/config.go

```diff
@@ -48,9 +58,30 @@ type Config struct {
 
 	Params map[string]string `yaml:"params"`
 
+	AutoMigration  *bool            `yaml:"autoMigration"` // If set to false, no tables will be created
+	DivisionPolicy DivisionPolicy   `yaml:"divisionPolicy"`
+	Mapper         []ResourceMapper `yaml:"mapper"` // Only DivisionPolicy is DivisionPolicyCustom it need to specify the mapping between resource and table
```

Comment:
Custom Policy related functionality is not being developed here, so it is recommended that it be removed to avoid binding/incompatible changes to subsequent pr's of this functionality.

Reply:
ok, removed